packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -36,22 +37,30 @@ const getWidth = (p: DropdownPropsAndState, v: DropdownVariables): string => {
   return v.width;
 };
 
+const isEmpty = prop => {
+  return (
+    typeof prop === 'object' &&
+    //@ts-ignore
+    !prop.props &&
+    !_.get(prop, 'children') &&
+    !_.get(prop, 'content')
```

Comment:
```suggestion
    prop.props &&
    _.get(prop, 'children') &&
    _.get(prop, 'content')
```

Reply:
when I tried to change it the suggestion then "default indicator" was not displayed:
![image](https://user-images.githubusercontent.com/22620150/84162825-1ba44c00-aa71-11ea-8eba-4416c785d51d.png)

because what we want to reach here is:
 if it is object verify it doesn't 
has any 'prop.props' what is this example: `<div>clear<div>`
has any children what is this example: `{{ children: 'clear' }}`
has any content what is this example: `{{ content: 'clear' }}`

Basically your proposal will never has "true" value. I will rather stick with our condition.